plugin/driver/pg_helper.go

```diff
@@ -359,6 +359,7 @@ func AddMetaconfig(domainID string, netID string, deviceID string, endpointID st
 		if device == deviceID {
```

Comment:
// no need to iterate
element, found := tm_data["workloads"].(map[string]interface{})[deviceID]